internal/task/task/common/sync_config.go

```diff
@@ -91,11 +91,11 @@ func newToolV2Mutate(dc *topology.DeployConfig, delimiter string, forceRender bo
 
 		// replace variable
 		value, err = dc.GetVariables().Rendering(value)
-		if err != nil {
-			return
+		if err == nil {
+			out = fmt.Sprintf("%s%s %s", key, delimiter, value)
+		} else { // FIX : add replace warring log
+			err = nil
```

Comment:
Why make the error nil？

Reply:
When I don't have a snapshot service configured, the Rendering function will introduce an error as shown in the following：
<img width="339" alt="image" src="https://github.com/opencurve/curveadm/assets/35839437/b4699607-c326-49cf-8774-e333faacca84">

I am aware that the current approach to handling this issue is not particularly elegant. Do you have any valuable suggestions for improvement?

Reply:
Maybe you can add a check before it, or talk it with your mentor or  @Cyber-SiKu for a better  approach.